PlayTools/PlaySettings.swift

```diff
@@ -45,16 +45,20 @@ let settings = PlaySettings.shared
         switch settingsData.iosDeviceModel {
         case "iPad6,7":
             return "J98aAP"
+        case "iPad7,1":
```

Comment:
There's no need to add more ipad models, this will generate more confusion among normal users. plus wont really help them

Reply:
More options would allow more granular performance tuning, especially users playing on fanless MacBook Airs. The 4th gen A12Z iPad model that is not available might provide better graphical quality than the 3rd gen A12X one without as high of a performance cost as the M1 option. 

For instance, after the resolution fix, Genshin is no longer playable with the same high in-game settings as before when using M1 identifier on M1 MacBook Air. The laptop hits its thermal limit much faster and throttles down. A12X is an option but at the cost of graphical quality and there is still some thermal headroom to play around with for MBA.

I don't think it would confuse users since they can simply consult the documentation to better understand its usage. Also if they are accessing and tinkering with app settings, it's safe to assume they know what they're doing.